### PR TITLE
Add 'How to install' and 'workspace' section into Tutorial.md

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This adds 'How to install' and 'workspace' section into Tutorial.md.

ONE-vscode-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

File: docs/Tutorial.md [hunk 1]
```diff
@@ -12,6 +12,36 @@
...
+## How to install ONE-vscode
+
+### Install vscode
+
+ONE-vscode is an [extension](https://code.visualstudio.com/docs/editor/extension-marketplace) of [vscode](https://code.visualstudio.com/). To use ONE-vscode, you should install vscode first.
+
+### Development environment
+
+ONE-vscode supports the following development environments:
+1. A single Ubuntu machie (to run vscode and ONE toolchains) or
```
Comment (llFreetimell) [inline docs/Tutorial.md:24]:
```suggestion
1. A single Ubuntu machine (to run vscode and ONE toolchains) or
```

File: docs/Tutorial.md [hunk 2]
```diff
@@ -37,6 +67,17 @@ ONE-vscode has 6 screen components.
 
 <!-- Rearrange content below -->
 
+## Workspace
+
+You need to open a folder (`File > Open Folder` menu) to start with ONE-vscode. After opening a folder, a [_workspace_ is constructed by vscode.](https://code.visualstudio.com/docs/editor/workspaces).
+
+The folder for ONE-vscode has the following files:
+- Your model files
```
Comment (llFreetimell) [inline docs/Tutorial.md:75]: `.circle` and profiling related files are excluded intentionally?

File: docs/Tutorial.md (hunk 1 above)
Comment (hyunsik-yoon) [reply docs/Tutorial.md]: Thanks. fixed.

Comment (hyunsik-yoon) [reply docs/Tutorial.md]: Fixed. Thanks.

File: docs/Tutorial.md (hunk 2 above)
Comment (hyunsik-yoon) [reply docs/Tutorial.md:75]: I think model file (and maybe ONE file) is the most important for readers. Intermediate files are actually not a concern for them. So I didn't mentioned those files. 
(In gcc tutorial, maybe source code and a.out would be mentioned while there are many other intermediate files)
For json file, I will explain in Profiling section later. :-)

**Review by llFreetimell — APPROVED** [review]
LGTM

File: docs/Tutorial.md [hunk 3]
```diff
@@ -12,6 +12,36 @@
...
+1. A single Ubuntu machine (to run vscode and ONE toolchains)
+2. A Windows machine (to run vscode) and a Ubuntu machine (to run ONE toolchains), connected with SSH extension
+
+A Ubuntu machine is mandatory because [ONE](https://github.com/Samsung/ONE) is officially released on Ubuntu. If your development environment is vscode on Windows, you need additional Ubuntu machine to run ONE toolchain. In additon, you need to [install SSH extension in vscode](https://code.visualstudio.com/docs/remote/ssh-tutorial).
+
+### Install ONE-vscode extension
+
+You can download the official extension package at [ONE-vscode release page](https://github.com/Samsung/ONE-vscode/releases), of which file extension is `*.vsix`.
+
+Then, install ONE-vscode extension by [following this instruction](https://code.visualstudio.com/docs/editor/extension-marketplace#_install-from-a-vsix).
```
Comment (yunjayh) [inline docs/Tutorial.md:33]: :+1:

**Review by yunjayh — APPROVED** [review]
LGTM

**Pull request merged**